docs: IA Phase 8: add the "Migrate to docs-as-code" section (#696)
Promptless/docs: src/content/docs/docs/migrate/choose-a-platform.mdx

---
title: Choose a platform
description: Pick a Git-backed documentation platform Promptless supports, weighing the tradeoffs between hosted docs tools and open-source static site generators
slug: docs/migrate/choose-a-platform
sidebar:
  hidden: false
  order: 38
  label: Choose a platform
---
import Info from '@components/fern/Info.astro';
import Tip from '@components/fern/Tip.astro';

> The right target platform is any Git-backed one that fits how your team works. Promptless publishes to a broad set of documentation platforms as long as your content lives in a GitHub repository, so the decision comes down to your team's needs rather than Promptless compatibility.

Reviewer: ```suggestion
The right target docs platform is any Git-backed one that fits how your team works. Promptless supports any documentation platform as long as your content lives in a GitHub repository, so the decision comes down to your team's needs rather than Promptless compatibility.
```

> 

Reviewer: ```suggestion
<Aside type="tip" icon="question-circle" title="Do I have to use Markdown?">
  No. Promptless can suggest updates to any text-based file format, including DITA XML, reStructuredText, AsciiDoc, and just about anything else.
</Aside>

```

## What Promptless supports

> Promptless doesn't require a specific platform — it requires that your content live as files in a GitHub repository it can open pull requests against. Any docs-as-code platform that builds from a Git-backed source works, whether that's a static site generator you host yourself or a hosted docs tool that keeps its content in a repository. Some platforms aren't natively Git-based but sync or mirror their content to GitHub, and those setups are fully supported too — Promptless only needs to reach the source through a repository.

Reviewer: ```suggestion
Promptless doesn't require a specific platform — it requires that your content live as files in a GitHub repository it can open pull requests against. Any docs-as-code platform that builds from a Git-backed source works, whether that's a static site generator you host yourself or a hosted docs tool that keeps its content in a repository. Some platforms aren't natively Git-based but sync or mirror their content to GitHub, and those setups are fully supported too. Promptless only needs to reach the source through a repository.
```

<Info>

> For the platforms teams most often run on Promptless, see the [supported platforms](/docs/connect/doc-locations/github-repos#supported-platforms) section of Doc locations & collections. If you're not sure whether your platform is supported, contact [help@gopromptless.ai](mailto:help@gopromptless.ai) — support is expanding based on user feedback, and the team can help you choose and set up a docs platform.

Reviewer: ```suggestion
For the platforms teams most often run on Promptless, see the [supported platforms](/docs/connect/doc-locations/github-repos#supported-platforms) section of Doc locations & collections. If you're not sure whether your platform is supported, contact [help@gopromptless.ai](mailto:help@gopromptless.ai). Support expands based on user feedback, and the team can help you choose and set up a docs platform.
```

</Info>

## How to choose

Every supported platform works with Promptless, so weigh them on the factors that matter to your team:

> - **Hosted docs tools vs. open-source generators.** Hosted, docs-focused platforms like Mintlify, Fern, GitBook, and ReadMe pair a managed publishing experience with Git-backed content. Open-source static site generators like Docusaurus, Starlight, MkDocs, Hugo, Nextra, and Vocs give you full control over hosting and theming and no per-seat cost, at the price of owning the build and deploy yourself.

Reviewer: ```suggestion
- **Hosted docs tools vs. open-source generators.** Hosted, docs-focused platforms like Mintlify, Fern, GitBook, and ReadMe pair a managed publishing experience with Git-backed content. Open-source static site generators like Starlight, Docusaurus, MkDocs, Hugo, Nextra, and Vocs give you full control over hosting and theming and no per-seat cost, at the price of owning the build and deploy yourself.
```

> - **Content format.** Most static site generators build from Markdown or MDX, though structured-authoring toolchains (DITA, reStructuredText, MadCap Flare) publish from their own source formats. If your team already authors in the target's format, the migration is mostly moving files; if you're coming from a WYSIWYG editor or a different source format, expect a conversion step (see [Migrate to a new docs platform](/docs/migrate/migrate-to-a-new-platform)).

Reviewer: ```suggestion
- **Content format.** Most static site generators build from Markdown or MDX, though structured-authoring toolchains publish from their own source formats. If your team already authors in the target's format, the migration is mostly moving files; if you're coming from a WYSIWYG editor or a different source format, expect a conversion step (see [Migrate to a new docs platform](/docs/migrate/migrate-to-a-new-platform)).
```

---
title: Choose a platform
description: Pick a Git-backed documentation platform Promptless supports, weighing the tradeoffs between hosted docs tools and open-source static site generators
slug: docs/migrate/choose-a-platform
sidebar:
  hidden: false
  order: 38
  label: Choose a platform
---
import Info from '@components/fern/Info.astro';
import Tip from '@components/fern/Tip.astro';

The right target platform is any Git-backed one that fits how your team works. Promptless publishes to a broad set of documentation platforms as long as your content lives in a GitHub repository, so the decision comes down to your team's needs rather than Promptless compatibility.

## Platforms Promptless supports

When your documentation lives in GitHub, Promptless can publish to any of these platforms:

- **Fern**
- **Mintlify**
- **ReadMe** (Refactored)
- **GitBook**
- **Docusaurus**
- **MkDocs**
- **Hugo**
- **Ghost**
- **Nextra**
- **Starlight** (Astro)
- **Vocs**

> - **Custom platforms** — any setup where the source lives in a repository, whatever the authoring format

Reviewer: Trim this to focus on the content instead of having a hard list of platforms, because we can support anything that's git-backed.

<Info>
This list is maintained on the [Doc locations & collections](/docs/connect/doc-locations/github-repos#supported-platforms) page. If the platform you want isn't listed, contact [help@gopromptless.ai](mailto:help@gopromptless.ai) — support is expanding based on user feedback, and the team can help you choose and set up a new docs platform.

> </Info>

Reviewer: Revise to read something like "If you're not sure if your platform is supported, contact..."
